gatsby-config.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
       // keep as first gatsby-source-filesystem plugin for gatsby image support
       resolve: 'gatsby-source-filesystem',
       options: {
-        path: `${__dirname}/static/images`,
+        path: `${__dirname}/static/img`,
```

Comment:
But I like `images` :( Our browsers are not fearful of URL characters. 

I know this was for consistency though. Kind of wish we could drag the consistency the other way.

Reply:
Okay!! 😆Will take care of it a coming PR.